package.json

```diff
@@ -50,5 +50,12 @@
       "jest": true,
       "node": true
     }
+  },
+  "jest": {
+    "testURL": "http://localhost"
+  },
+  "prettier": {
```

Comment:
Could you please remove `prettier`? Our ESLint configuration is enough for 'opinionated' formatting :)

Reply:
done!